### ISSUE TITLE
Move relative objects to Otter.Core namespace

### ISSUE THREAD

**Issue by WillSams** [WillSams/Otter#6]
In an effort to move everything out of the general "Otter" namespace, all base objects in .cs files under the Otter/Core path should be moved to Otter.Core namespace.

No specs (tests) need to be written for this task, just clean & rebuild project and run FlippyFlop to test if the changes did not cause an adverse impact.

If you want to contribute:
- Make a fork of the repository.
- Switch to the test-otter-epic branch and do a git pull.
- Make your changes and commit.
- Submit a pull request.

Once the pull request is approved, I'll merge your code into the test-otter-epic branch.  Code from test-otter-epic branch will be merged into dotnetcore-dev once all stories within the epic are completed.
